File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int szs=s.size();
        int szt=s.size();
        if(szs!=szt)
        {
            return false;

        }
        vector<int>mp(26);
        for(char&ch:s)
        {
            ++mp[ch-'a'];
        }
        for(char&ch:t)
        {
            --mp[ch-'a'];
        }
        for(int&it:mp)
        {
            if(it!=0)return false;
        }
        return 1;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int szs=s.length();
        int szt=t.length();
        if(szs!=szt)
        {
            return false;

        }
        int[] mp = new int[26];
        for (char ch : s.toCharArray())
        {
            mp[ch - 'a']++;
        }
        for (char ch : t.toCharArray()) 
        {
            mp[ch - 'a']--;
        }
        for (int it : mp)
        {
            if (it != 0)
            return false;
        }
        return true;
    }
}

        